File: sdkwork-app-sdk-csharp/Models/OrderListQueryForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class OrderListQueryForm
    {
        public int? Page { get; set; }
        public int? Size { get; set; }
        public string? Status { get; set; }
        public string? OrderType { get; set; }
        public string? Keyword { get; set; }
        public string? StartTime { get; set; }
        public string? EndTime { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/order_list_query_form.go
package types

// 查询订单列表请求
type OrderListQueryForm struct {
	Page int `json:"page"`
	Size int `json:"size"`
	Status string `json:"status"`
	OrderType string `json:"orderType"`
	Keyword string `json:"keyword"`
	StartTime string `json:"startTime"`
	EndTime string `json:"endTime"`
}
